ffpass/__init__.py

```diff
@@ -274,23 +454,30 @@ def addNewLogins(key, jsonLogins, logins):
     jsonLogins["nextId"] += len(logins)
 
 
-def guessDir():
-    dirs = {
-        "darwin": "~/Library/Application Support/Firefox/Profiles",
-        "linux": "~/.mozilla/firefox",
-        "win32": os.path.expandvars(r"%LOCALAPPDATA%\Mozilla\Firefox\Profiles"),
```

Comment:
This did not work for me on Windows 10 nor Hiren's USB. It's possible though we may need logic for supporting/checking both local and roaming. I am a Linux user and defer to your knowledge on this matter.